README.md

```diff
@@ -1 +1,44 @@
 
+# Project Name
+
+To Do List Application
+
+# PR
+
+1-Add items to the to do list
+2-Checkbox input form 
+![screenshot](./Capture.png)
```

Comment:
- [OPTIONAL] The link to the screenshot image is broken, please kindly fix that, as it will make your readme file look more professional.